wrangler.toml

```diff
@@ -53,9 +54,12 @@ MAX_SHARDS = "825"
 FF_RATE_LIMITER_ENABLED = "false"
 FF_EGRESS_TRACKER_ENABLED = "true"
 FF_TELEMETRY_ENABLED = "true"
-TELEMETRY_RATIO = 0.0005
 FF_DELEGATIONS_STORAGE_ENABLED = "true"
 FF_RAMP_UP_PROBABILITY = "0"
+FF_DAGPB_CONTENT_CACHE_TTL_SECONDS = 86400
```

Comment:
@alanshaw suggested making it 30 days?

Reply:
That's right. Just fixed. Thanks for flagging.